cassovary-core/src/main/scala/com/twitter/cassovary/graph/Graph.scala

```diff
@@ -25,4 +27,6 @@ trait Graph {
   def getNodeById(id: Int): Option[Node]
 
   def existsNodeId(id: Int) = getNodeById(id).isDefined
+
+  val nodeLabels = new Labels[Int] //empty to start with
```

Comment:
I'm wondering if it wouldn't be better to add labels as a trait and keep the possibility to have a graph with no labels.